x-pack/plugins/security_solution/public/detection_engine/rule_management/logic/coverage_overview/build_coverage_overview_mitre_graph.ts

```diff
@@ -66,10 +66,30 @@ export function buildCoverageOverviewMitreGraph(
       }
     }
   }
+  const tacticOrder = [
+    'TA0043',
+    'TA0042',
+    'TA0001',
+    'TA0002',
+    'TA0003',
+    'TA0004',
+    'TA0005',
+    'TA0006',
+    'TA0007',
+    'TA0008',
+    'TA0009',
+    'TA0011',
+    'TA0010',
+    'TA0040',
+  ];
```

Comment:
This should probably live somewhere else. Perhaps a `constants.ts` file in the same coverage overview logic folder?

Reply:
It's probably ok to keep it in this file. Let's just move it outside of the `buildCoverageOverviewMitreGraph` function to avoid creating the array on each function call.

If we want to further refactor this, I'd say column order is a UI logic (not a business logic) => we should keep it close to or in the grid component. But I would probably not suggest doing it in this PR.